docs/front-end-overview.md

```diff
@@ -183,3 +183,96 @@ The part within the callback would execute once the GET request had returned, wi
+
+ENCODE implementation of [StyleLint](https://stylelint.io/)  uses [Stylelint Config Standard](https://github.com/stylelint/stylelint-config-standard) rules collection; although some exceptions are made to the rules. The rules can  be found by examining [https://github.com/ENCODE-DCC/encoded/blob/dev/.stylelint.json](https://github.com/ENCODE-DCC/encoded/blob/dev/.stylelint.json),  under the "Rules" section. The rules should not be disabled without consulting other front-end developers. There is rarely ever a need to disable any rule and almost always there are legitimate workarounds. The current disabled rules are [design debt](https://en.wikipedia.org/wiki/Technical_debt).
+
+It is important to make sure new style sheets  satisifies [StyleLint](https://stylelint.io/) rules, otherwise, the CircleCI under  (ci/circleci: stylelint) check will fail the build.
+
+You can check for and correct [StyleLint](https://stylelint.io/) violations either through the command line or with a plugin to a text editor. Either way suffices as long  as violations are corrected.
+
+### Command line
+
+#### Determining [StyleLint](https://stylelint.io/)  Violations
+
+The  format for checking for [StyleLint](https://stylelint.io/) violations is:
+`./node_modules/stylelint/bin/stylelint.js <file or folder path>`
+
+For example, to find all rule violations in a folder, you can run:
+`./node_modules/stylelint/bin/stylelint.js src/encoded/static/scss/`
+
+You can also check just one file (e.g datacolor.js) by  running:
+`./node_modules/stylelint/bin/eslint.js src/encoded/static/scss/style.scss`
+
+#### Correcting [StyleLint](https://stylelint.io/) Violations
+
+The format for correcting the [StyleLint](https://stylelint.io/) violations is:
+`./node_modules/stylelint/bin/stylelint.js <file or folder path> --fix`
+
+For example, you fix all rule violations in a folder, by running:
+`./node_modules/stylelint/bin/stylelint.js src/encoded/static/scss/ --fix`
+
+You can also fix violation for just one file (e.g style.css) by  running:
+`./node_modules/stylelint/bin/stylelint.js src/encoded/static/components/style.scss -- fix`
+
+Numerous other options can be found at [StyleLint Command line interface Documention page](https://stylelint.io/user-guide/usage/cli)
+
+You can search on [YouTube](https://www.youtube.com/results?search_query=stylelint+) to see if there are any videos that illustrate using the command line with  [StyleLint](https://stylelint.io/).  In general, the idea is similar to  the [ESLint](https://eslint.org/).
+#### Editor Extension
+
+Widely used code text editors generally have extensions that allows you to visually detect [StyleLint Command line interface Documention page](https://stylelint.io/user-guide/usage/cli) violations by seeing a squiggly under the code in question. They also display information on what rule was violated by mousing over the squiggly and sometimes automated correction by right clicking and selecting an appropiate option.
```

Comment:
"allows" should be "allow"

Reply:
"allow you to detect by seeing" is sort of wordy construction, you could just say "Widely used code text editors have extensions that highlight code violations with underlines/emphasis, and often display information about rule violations and sometimes automated corrections on mouseover...."

Reply:
"appropriate" not "appropiate"